src/reporting/branch.se

```diff
@@ -139,3 +139,13 @@ def isContainerForReportingToken(shadyReportingToken: address):
 def isParentOf(shadyChild: address):
     branchId = shadyChild.getParentPayoutDistributionHash()
     return(self.childBranches[branchId] == shadyChild)
+
+def isContainerForShareToken(denominationToken):
```

Comment:
Parameter name should be `shadyShareToken`, since that is what we assume it will be in this context (shareToken).  The `shady` prefix is the way help make it clear to readers that the input is untrusted.